kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml

```diff
@@ -165,6 +165,11 @@ domainHomeImageBase: container-registry.oracle.com/middleware/weblogic:12.2.1.4
 # Default is /tmp/dhii-sample/tools
 # toolsDir: /tmp/dhii-sample/tools
 
+# WLST script that the create domain script uses to create a WebLogic domain
+# when using wlst mode.
+# This value is ignored when WDT is used in creating a WebLogic domain.
+createDomainWlstScript: wlst/create-wls-domain.py
```

Comment:
is there an equivalent way to pass in the desired WDT models/archive? (customers will ask)